### ISSUE TITLE
FetchCharacterAchievements() is returning an error if achievements are private

### ISSUE THREAD

**Issue by bourgeoisor** [xivapi/godestone#15]
**Describe the bug**
I was under the impression that `FetchCharacterAchievements()` was not supposed to panic under a 403 (achievements private in Lodestone) and simply set a field in the returning `AllAchievementInfo` object, but I am instead seeing a nil `AllAchievementInfo`, as well as a Forbidden error returned. I am taking this expectation from this comment: https://github.com/xivapi/godestone/blob/main/scraper.go#L223

**To Reproduce**
```
s = godestone.NewScraper(bingode.New(), godestone.EN)
id := 35782373
achievements, allAchievementsInfo, err := s.FetchCharacterAchievements(uint32(id))
fmt.Println(achievements)
fmt.Println(allAchievementsInfo)
fmt.Println(err)
```

**Expected behavior**
I expected `AllAchievementInfo` to be non-nil, and `error` to be nil to be consistent with the function comment.

**Console output**
```
[]
<nil>
Forbidden
```

**Desktop (please complete the following information):**
 - OS: Windows 10
 - Version 19043.1237

**Additional context**
Version of godestone used: `v2.1.1`

**Comment by karashiiro** [comment]
Seems to be a typo [here](https://github.com/xivapi/godestone/blob/main/scraper.go#L236), fixing.

**Comment by karashiiro** [comment]
Oh, it's because I just copied the function above it and forgot to change that bit 😓

**Comment by bourgeoisor** [comment]
Sweet, thank you! Closing this issue.

**Comment by karashiiro** [comment]
Released v2.1.2
